src/AppMenuSearch.cc

```
@@ -0,0 +1,386 @@
+            if (safeAction) {
+                safeAction->trigger();
+            }
+            if (searchMenu) {
+                searchMenu->hide();
+            }
+        });
+        searchMenu->addAction(newAction);
+        resultCount++;
+    }
+
+    Q_EMIT repositionRequested();
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)